### PR TITLE
FFM-7362 Additional Headers

### PR DESCRIPTION
**Pull request by erdirowlands**
# What
Adds the following headers to all endpoints (except `auth`) for tracking usage

Harness-SDK-Info
Harness-EnvironmentID
Harness-AccountID

# Testing
Manual using local proxy that prints out headers for each request that came through.

### CODE REVIEW

File: featureflags/client.py [hunk 1]
```diff
@@ -127,7 +127,13 @@ def authenticate(self):
...
             }
         )
-        self._client.with_headers({"User-Agent": "PythonSDK/" + VERSION})
+        # Additional headers used to track usage
+        additional_headers = {
+            "User-Agent": "PythonSDK/" + VERSION,
+            "Harness-SDK-Info": f'Python {VERSION} Server',
+            "Harness-EnvironmentID": self._environment_id,
+            "Harness-AccountID": decoded["accountID"]}
+        self._client = self._client.with_headers(additional_headers)
```
Comment (erdirowlands) [inline featureflags/client.py:136]: The return value of this wasn't actually being used previously, so any custom headers would not have been set.

**Review by chrisblakely01 — APPROVED** [review]
LGTM just added a Q

**Review by andybharness — APPROVED** [review]

File: featureflags/client.py [hunk 2]
```diff
@@ -127,7 +127,13 @@ def authenticate(self):
                 'cluster': self._cluster
             }
         )
-        self._client.with_headers({"User-Agent": "PythonSDK/" + VERSION})
+        # Additional headers used to track usage
+        additional_headers = {
+            "User-Agent": "PythonSDK/" + VERSION,
+            "Harness-SDK-Info": f'Python {VERSION} Server',
+            "Harness-EnvironmentID": self._environment_id,
+            "Harness-AccountID": decoded["accountID"]}
```
[the remark this answers is not in the document]
Comment (erdirowlands) [reply featureflags/client.py:135]: Yep it'll be a runtime exception. But yes the accountID will always be on the token. We could use `get()` and provide a default value, or check if the key exists in the dictionary first, but I don't personally think it's required.

**Comment by erdirowlands** [comment]
Merging despite red build - flaky sse test.

**Pull request merged**